src/util/handlebars/handlebar-helpers.ts

```diff
@@ -4,9 +4,13 @@ import { isPlainObject } from "lodash";
 export const concatStringHelper = (...strings: string[]) => strings.filter((arg: unknown) => typeof arg !== "object").join(" ");
 export const toLowercaseHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().toLowerCase() || "";
 export const replaceSpaceWithHyphenHelper = (str?: string) => !isPlainObject(str) && str?.toString?.().replace(/ /g, "-") || "";
+export const logicalOrHelper = (...args: unknown[]) => args.filter((arg: unknown) => typeof arg !== "object").some(Boolean);
+export const logicalAndHelper = (...args: unknown[]) => args.filter((arg: unknown) => typeof arg !== "object").every(Boolean);
```

Comment:
I'm not sure how I feel about these.  I understand what you're trying to do, but you're now adding complex logic in handlebars with no compilation errors, which is what we're trying to prevent with typescript.

I think this will encourage bad coding practices in handlebar files.  Could we please remove them and instead, if you need to do complex logic, do it beforehand hand and pass it in as a config property to the template.  If you type it correctly, we'll be able to catch issues before they're a problem :)

Reply:
This helpers have been removed and the values needed have been passed within the configuration for the view.